mltsp/custom_feature_tools.py

```diff
@@ -460,12 +470,12 @@ def extract_feats_in_docker_container(container_name, path_to_tmp_dir):
         with open(os.path.join(path_to_tmp_dir, "results_list_of_dict.pkl"),
                   "rb") as f:
             results_list_of_dict = pickle.load(f)
+            return results_list_of_dict
     except:
```

Comment:
`try: 
    ...
except:
    raise
finally:
   # I want this to happen even if an exception is raised in the try block`